KomaMRIBase/src/datatypes/sequence/ADC.jl

```diff
@@ -39,9 +39,11 @@ mutable struct ADC
     end
 end
 
-# ADC comparison
+"""
+For comparing two `ADC`s custom types
+"""
 Base.isapprox(adc1::ADC, adc2::ADC) = begin
-    return all(length(getfield(adc1, k)) ≈ length(getfield(adc2, k)) for k ∈ fieldnames(ADC))
+    return all(length(getfield(adc1, k)) ≈ length(getfield(adc2, k)) for k ∈ fieldnames(ADC)) &&
         all(getfield(adc1, k) ≈ getfield(adc2, k) for k ∈ fieldnames(ADC))
```

Comment:
Not all of the functions need to be in one line, separate to improve readability.